django/thunderstore/community/tests/test_package_listing.py

```diff
@@ -121,14 +122,14 @@ def test_package_listing_community_read_only(
     active_package_listing: PackageListing,
 ) -> None:
     c = Community.objects.create(name="Test Community")
+    active_package_listing.community = c
     with pytest.raises(ValidationError) as exc:
-        active_package_listing.community = c
         active_package_listing.save()
     assert "PackageListing.community is read only" in str(exc.value)
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize("require_approval", (False, True))
+@pytest.mark.parametrize("require_approval", [False, True])
```

Comment:
Not a blocker, but what's up with these changes? Tuples are a better data type for immutable data than lists

Reply:
Changed based on Flake8 warnings. Fixed by [prepending this stack with a PR](https://github.com/thunderstore-io/Thunderstore/pull/914) that configs Flake8 to prefer tuples.